WcaOnRails/spec/ruby_version_spec.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe "The ruby version used" do
+  it "matches the one in the .ruby-version file" do
```

Comment:
I don't think we need to check Travis.
E.g. we update `.ruby-version` to contain `2.4.0`, then we make a PR and Travis runs. It's configuration file contains `2.3.1`, which is outdated. The point is that the test I've just added runs for Travis as well and should fail. Right?